ui/src/components/Issuer/IssuedToken.tsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react'
 import { useParams } from 'react-router-dom'
-import { Header, List } from 'semantic-ui-react'
+import { Header, Table } from 'semantic-ui-react'
 
 import { useStreamQueries } from '@daml/react'
 import { Token } from '@daml.js/da-marketplace/lib/Marketplace/Token'
+import { AssetDeposit } from '@daml.js/da-marketplace/lib/DA/Finance/Asset'
+import { makeContractInfo } from '../common/damlTypes'
 
 import Page from '../common/Page'
 import PageSection from '../common/PageSection'
+import ReactJson from 'react-json-view'
```

Comment:
It doesn't look like this got added into the `package.json` -- did you use `yarn install react-json-view`?